### PR TITLE
[15.0][MIG] l10n_fr_ecotaxe

### PR DESCRIPTION
**Pull request by SilvioC2C**

### CODE REVIEW

File: l10n_fr_ecotaxe/models/account_ecotaxe_classification.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        "des EEE provenant de pays hors Union Européenne"
+        "DIS ==> Vendeur à distance : est établie dans un autre Etat\n"
+        "membre ou dans un pays tiers et vend en France des EEE par\n"
+        "communication à distance",
+    )
+    ecotaxe_deb_code = fields.Char()
+    ecotaxe_scale_code = fields.Char()
+
+    @api.onchange("ecotaxe_type")
+    def onchange_ecotaxe_type(self):
```
Comment (ivantodorovich) [inline l10n_fr_ecotaxe/models/account_ecotaxe_classification.py]:
```suggestion
    def _onchange_ecotaxe_type(self):
```

File: l10n_fr_ecotaxe/views/product_template_view.xml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    <record id="product_template_form_ecotaxe" model="ir.ui.view">
+        <field name="name">product.template.ecotaxe.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <xpath expr="//label[@for='list_price']" position="before">
+                <field name="ecotaxe_classification_id" />
+                <field
+                    name="ecotaxe_amount"
+                    widget="monetary"
```
Comment (ivantodorovich) [inline l10n_fr_ecotaxe/views/product_template_view.xml]: I'd change this field type to `Monetary` on the python file, then there's no need to specify `widget="monetary"`
(same for the other cases)

File: l10n_fr_ecotaxe/models/account_move_line.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+#   @author Mourad EL HADJ MIMOUNE <mourad.elhadj.mimoune@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class AcountMoveLine(models.Model):
+    _inherit = "account.move.line"
+
+    subtotal_ecotaxe = fields.Float(store=True, compute="_compute_ecotaxe")
```
Comment (ivantodorovich) [inline l10n_fr_ecotaxe/models/account_move_line.py:11]: Better to use `Monetary` fields here

**Comment by ivantodorovich** [comment]
@OCA/local-france-maintainers could you approve the github workflow run? 🙏🏻

**Review by ivantodorovich — APPROVED** [review]

**Review by ajaniszewska-dev — APPROVED** [review]

**Comment by ajaniszewska-dev** [comment]
@SilvioC2C  any chance to merge this PR? I got po file to update translation but cant do it through odoo community translation page because this PR isn't merged so officially module doesn't exist on 15.0 yet.

**Comment by SilvioC2C** [comment]
@ajaniszewska-dev I'd love to, but unluckily I don't have merge rights :sweat_smile:

**Review by gurneyalex — APPROVED** [review]

**Comment by gurneyalex** [comment]
@SilvioC2C can you check the merge issues?

**Comment by SilvioC2C** [comment]
@gurneyalex seems a pre-commit issue, if I'm reading [this](https://github.com/OCA/l10n-france/commits/f2f2313a2cfdc8f7d6226b17172dc9c14644411d) and [this](https://github.com/OCA/l10n-france/runs/5405653620?check_suite_focus=true) correctly.
I'll take care of this asap.

**Comment by SilvioC2C** [comment]
@gurneyalex issue should be solved, mind trying to merge again when you're available?

**Pull request merged**